packages/react-liveness/src/components/FaceLivenessDetector/service/machine/machine.ts

```diff
@@ -334,6 +334,8 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
               100: { target: 'checkRecordingStarted' },
             },
           },
+          // Evaluates face match and moves to checkMatch
```

Comment:
Comments were part of https://github.com/aws-amplify/amplify-ui/pull/5093 but think they got deleted somehow so adding back/updating for the pause